_docs/resources/sketch-library.md

```diff
@@ -2,8 +2,9 @@
 title: Install the Helix Sketch Library
 layout: docs
 status: stable
-resource: true
+resource: false
 category: Resources
+parent: Libraries & Repo's
```

Comment:
I take it we'll use the `parent` call at some point for the Resources section? Or is this just to add it in case?

Reply:
Setting the parent to Libraries & Repos enables us to highlight that nav item, while still keeping the child page (sketch-library.md) invisible from the left nav.  This is how the Text Conventions articles are handled as well.

When we move to one site, we should consider whether we might need a tertiary level of navigation for things like this.